### ISSUE TITLE
Starts Crashing upon i try playing a video or a short

### ISSUE THREAD

**Issue by JudeTheTechSavvyDude** [arichornlover/YouTubeRebornPlus#112]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I read the FAQ

### Device info

- iOS/iPadOS version:17.4.1
- Device model:iPhone 13
- Sideload tool (AltStore, SideStore, Sideloadly, TrollStore,...): Sideloadly
- The specific version of YouTubeRebornPlus (**latest** or **newest** is **NOT** a version number!):19.10.7

### Describe the bug

When i play a video it instantly crashes and every video i look up it also crashes

### Steps to reproduce the issue

1. Play any video or shorts
2. It should start crashing 
3. 

### Are you using the newest version of YouTubeRebornPlus? If not, why?

✅ Yes, I'm using the latest version of YouTubeRebornPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, It doesn't

**Comment by rainerlogan** [comment]
Any fix for this? I’m getting this too out of nowhere

**Comment by rainerlogan** [comment]
Hey. I think I found a fix!

Try disabling iSponsorBlock entirely and then restarting the app. You can find it at the very too with the cog icon inside of the shield icon. Disable it entirely and then restart, see if that works. :)

**Comment by someperson** [comment]
There's a bug in iSponsorBlock causing this, though I've found a workaround: go to Settings > YouTubeRebornPlus > Version Spoofer Picker, and select "v17.49.6". After that, it works!

**Comment by someperson** [comment]
There's a bug in iSponsorBlock causing this, though I've found a workaround [based on this](https://github.com/Galactic-Dev/iSponsorBlock/issues/110#issuecomment-2023900010):

1. Go to your profile on the top right.
2. Go to Settings.
3. Go to YouTubeRebornPlus.
4. Enable App Version Spoofer.
5. Set the Version Spoofer Picker to v18.12.2.
6. Restart the app.

**Comment by jdactal** [comment]
Yeah, it seems to be an issue with iSponserBlock. It apparently should be fixed now, but then this app needs to be updated.

**Comment by Zaid-AlKhafaji** [comment]
Turn off iSponsorblock
